added documentation for translation review (#18)
oppia/creator-documentation: docs/reviewers/review.rst

.. _review:

> Review Rights

Reviewer: ```suggestion
Reviewing Lesson Contributions
```

================

> Reviewing is critical in ensuring that the translations, questions and voiceovers submitted by the contributors are correct and error-free. In order to review questions, translations or voiceovers, a user must be authorised by the admin.

Reviewer: Nit: For consistency, prefer North American English, i.e. "authorized" rather than "authorised".

Author: Done!

> The review rights a user has are listed in the header of the contributor-dashboard under the My Contributions tab. Review rights for translations and voiceovers are language specific whereas that for questions is not. The languages in which you can review translations and voiceovers are also listed in the header.

Reviewer: Grammar fixes and minor improvements:

```suggestion
The review rights of a user are listed in the header of the Contributor Dashboard under the **My Contributions** tab. Review rights for translations and voiceovers are language specific whereas those for questions are not. The languages in which you can review translations and voiceovers are also listed in the header.
```

Author: Done!
